sql/mob_skill_lists.sql

```diff
@@ -2692,3 +2692,770 @@ INSERT INTO `mob_skill_lists` VALUES ('Alexander_LowHP',785,2145); -- Gospel of
 INSERT INTO `mob_skill_lists` VALUES ('Alexander_LowHP',785,2146); -- Void of Repentance "Repent thy irreverence..."
 INSERT INTO `mob_skill_lists` VALUES ('Alexander_LowHP',785,2147); -- Divine Judgement   "Accept thy destruction..."
 
+-- Alkonost
+INSERT INTO `mob_skill_lists` VALUES ('Alkonost',786,576); -- back heel
+-- INSERT INTO `mob_skill_lists` VALUES ('Alkonost',786,579); -- choke breath
+INSERT INTO `mob_skill_lists` VALUES ('Alkonost',786,578); -- nihlity song
+INSERT INTO `mob_skill_lists` VALUES ('Alkonost',786,580); -- fantod
+-- INSERT INTO `mob_skill_lists` VALUES ('Alkonost',786,1330); -- hoof volley
```

Comment:
@Hozu I may have commented the wrong line by mistake I just put the fix in that will correct that issue.